tests/animal_module_tests/ration/test_amino_acid.py

```diff
@@ -0,0 +1,247 @@
+import pytest
+from pytest import approx
+
+from RUFAS.routines.animal.ration.amino_acid import (
+    AminoAcidCalculator,
+    EssentialAminoAcidRequirements,
+)
+
+
+@pytest.mark.parametrize(
+    "amino_acid, NPscurf, expected",
+    [
+        ("arginine", 1.2, 0.1152),
+        ("histidine", 1.2, 0.021),
+        ("isoleucine", 1.2, 0.03552),
+        ("leucine", 1.2, 0.08315999999999998),
+        ("lysine", 1.2, 0.06768),
+        ("methionine", 1.2, 0.0168),
+        ("phenylalanine", 1.2, 0.04332),
+        ("threonine", 1.2, 0.048119999999999996),
+        ("thryptophan", 1.2, 0.00876),
+        ("valine", 1.2, 0.05592),
+    ],
+)
+def test_calculate_scurf(amino_acid: str, NPscurf: float, expected: float) -> None:
+    """
+    Unit test for AminoAcidCalculator._calculate_scurf()
+    """
+    amino_acid_calculator = AminoAcidCalculator()
+    actual_result = amino_acid_calculator._calculate_scurf(amino_acid, NPscurf)
+
+    assert actual_result == approx(expected, abs=0.001)
+
+
+@pytest.mark.parametrize(
+    "amino_acid, body_weight, expected",
+    [
+        ("arginine", 256.7, 1.3155874999999997),
+        ("histidine", 256.7, 0.48772999999999994),
+        ("isoleucine", 256.7, 0.592014375),
+        ("leucine", 256.7, 1.3268181249999997),
+        ("lysine", 256.7, 1.26745625),
+        ("methionine", 256.7, 0.38023687500000003),
+        ("phenylalanine", 256.7, 0.707529375),
+        ("threonine", 256.7, 0.7765175),
+        ("thryptophan", 256.7, 0.16845937500000002),
+        ("valine", 256.7, 0.826253125),
+    ],
+)
+def test_calculate_endogenous_urinary_excretion(amino_acid: str, body_weight: float, expected: float) -> None:
+    """
+    Unit test for AminoAcidCalculator._calculate_endogenous_urinary_excretion()
+    """
+    amino_acid_calculator = AminoAcidCalculator()
+    actual_result = amino_acid_calculator._calculate_endogenous_urinary_excretion(amino_acid, body_weight)
+    assert actual_result == approx(expected, abs=0.001)
```

Comment:
```suggestion
    assert actual_result == approx(expected)
```
I Have tried this on a few tests and without setting the abs value it is still fine. The default is 1e-6, so keeping the default is also more accurate.